pyiso/caiso.py

```diff
@@ -486,7 +497,7 @@ def fetch_oasis(self, payload={}, return_all_files=False):
             return default_return_val
 
         # load xml into soup
-        soup = BeautifulSoup(content[0])  # Check the first file for errors
+        soup = BeautifulSoup(content[0], 'lxml')  # Check the first file for errors
```

Comment:
This will cause BeautifulSoup to throw an error if lxml is not installed rather than use a parser with potentially different behavior.  Is this desired behavior?  My thought was that we would rather get a clear error, than potentially an error somewhere else in processing that was dependent on the machine configuration.